extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -215,6 +220,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				}
 				this._onDidChangeTreeData.fire(undefined);
 			}
+			TelemetryReporter.createActionEvent(NotebookTelemetryView, NbTelemetryActions.CloseBook).send();
```

Comment:
FYI you can just call sendActionEvent to send it directly if you aren't adding additional properties